src/_mt2/mt2_Lallyver2.h

```diff
@@ -2,10 +2,29 @@
 /*                                                                                      */
 /*                  Finding Asymmetric MT2 by Characteristic Eq Cubic Discriminant    	*/
 /*        		                                                          				*/
-/*                  Copyright: Colin Lally                                              */
 /*                  December 28, 2015, ver2                                          	*/
 /*                                                                                    	*/
 /****************************************************************************************
+ 
+ * Copyright 2025 Colin Lally

```

Comment:
@kesterlester I have diverged from https://github.com/tpgillam/mt2/issues/84#issuecomment-2830256002 , insofar as I'm using 2025 rather than 2026 (since that's the current year :) )

Reply:
Good.